portfolio/src/main/webapp/images.html

```diff
@@ -0,0 +1,68 @@
+          <div class="image-label">
+            <a href="/images/TKDCornellTeam.jpg">
+              <img src="/images/TKDCornellTeam.jpg" alt="Cornell TKD Team Picture">
+            </a>
+            <p>This is my Taekwondo team!</p>
+          </div>
+          <div class="image-label">
+            <a href="/images/BlueBeltGroup.jpeg">
+              <img src="/images/BlueBeltGroup.jpeg" alt="Cornell Blue Belts Picture">
+            </a>
+            <p>These are my fellow blue belts!</p>
+          </div>
+          <p><strong>Fun Memories</strong></p>
+          <div class="image-label">
+            <a href="/images/WizardsUniteProfile.png">
+              <img src="/images/WizardsUniteProfile.png" alt="My Wizards Unite Profile">
+            </a>
+            <p>I am a huge Harry Potter fan!</p>
+          </div>
+          <div class="image-label">
+            <a href="/images/MangoMangoSweets.jpg">
+              <img src="/images/MangoMangoSweets.jpg" alt="Mango Mango dessert">
+            </a>
+            <p>Yum! I have a sweet tooth.</p>
+          </div>
+          <div class="image-label">
+            <a href="/images/MCStarterHouse.jpg">
+              <img src="/images/MCStarterHouse.jpg" alt="My first Minecraft starter house">
+            </a>
+            <p>My first starter house in Minecraft.</p>
+          </div>
+        </div>
+      </div>
+      <div class="section">
+        <h2>Why These Pictures?</h2>
+        <p>
+          I enjoy photographing and keeping pictures of memorable moments. I have pictures of the Taekwondo team that I am currently on 
+          as well as moments that I have with my college friends. The base of the photo with the Harry Potter references was taken by a good friend of mine, Andrea, who 
+          is an English major. I took the dessert photo at a restaurant called Mango Mango that recently opened in Ithaca Collegetown. 
+          I went there with another friend named Lily because we wanted to eat dessert after watching the movie JoJo Rabbit at Cornell Cinema.
+          The Minecraft photo was taken very recently, over Memorial weekend. This was my first time playing Minecraft and I didn't know what to do, 
+          so I texted Andrea for help because she knew how to play and she explained to me what I needed to do to get started. 
+          Surprisingly, I haven't had my first death yet, although the first skeleton that I encountered almost accomplished that. 
+          It was nice that I got to do something with Andrea even though we are both stuck at home so I took a picture of the starter house to show her what I created using her advice.
+          <br/><br/>
+          Sometimes I do take pictures of stuff other than my life such as nature. Examples are animals and beautiful landscapes such as Ithaca Falls and sunsets. 
+          I screenshot interesting quotes that I read on Instagram and interesting posts that I read on Facebook.
```

Comment:
nit: Better to keep the line within a certain length for ease of viewing in Github without overflow (horizontal scrolling)